cvmfs/util.h

```diff
@@ -274,7 +275,7 @@ class UniquePtr : SingleCopy {
 
   inline operator bool() const        { return IsValid(); }
   inline T& operator*() const         { return *ref_; }
-  inline UniquePtr& operator=(T* ref) { ref_ = ref; return *this; }
+  inline UniquePtr& operator=(T* ref) { delete ref_; ref_ = ref; return *this; }
```

Comment:
Ooops! Very good catch. Thanks. [Fixed](https://github.com/reneme/cvmfs/commit/1f6d5003876d5b3560db8a66d1801a4ba0a2740c) and [tested](https://github.com/reneme/cvmfs/commit/875ca799b275dabb3c4d972a1fdfd677aaa282a3).